src/main/flight/position.c

```diff
@@ -84,6 +84,7 @@ void calculateEstimatedAltitude(timeUs_t currentTimeUs)
     int32_t baroAlt = 0;
 
     int32_t gpsAlt = 0;
+    int16_t gpsVertSpeed = 0;
```

Comment:
All occurrences of `gpsVertSpeed` should be guarded by `USE_VARIO`, to stop it from breaking on targets that do not define `USE_GPS` or `USE_VARIO`.

Reply:
This is a local variable that will be optimized out by the compiler on targets that don't define USE_VARIO nor USE_GPS, as the variable won't be used at all. In case of USE_GPS and not USE_VARIO, the variable is assigned but never used, so I assume gcc will optimize it out as well (it should be internally flagged as dead code).

Reply:
Oh, I see now warnings are set as errors...